src/components/Buttons.js

```diff
@@ -0,0 +1,27 @@
+/* eslint-disable react/require-default-props */
+/* eslint-disable react/default-props-match-prop-types */
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const ButtonPanel = ({ buttonName, Color, Width }) => {
+  const ButtonName = buttonName;
+  let styles = { backgroundColor: Color, width: Width };
+  if (Width) {
+    styles = { backgroundColor: Color, width: '209%' };
+  }
+
+  return (
+    <button type="button" style={styles}>{ButtonName}</button>
+  );
```

Comment:
Here you should render all the buttons of the calculator using the `Button` component for every button.

Reply:
Thanks i will fix that issues thanks !!